content/pages/community.markdown

```diff
@@ -0,0 +1,40 @@
+Title: Community
+url: community/
+save_as: community/index.html
+
+Community
+---------
```

Comment:
This is an h2. The original page has h1 instead. Convert all headings in the markdown document to h1 as well.